include/seastar/http/httpd.hh

```diff
@@ -207,6 +212,25 @@ private:
     friend class http_server_tester;
 };
 
+template<typename Clock = std::chrono::steady_clock>
+future<> http_server::graceful_pre_stop(int timeout_ms) {
```

Comment:
Since you already use a Clock template, just use `Clock::duration` type for the timeout parameter instead of `int`.

Also, it would be better to add a concept for limiting the types to actual clocks - you can require this type to be a TrivialClock: https://en.cppreference.com/w/cpp/named_req/TrivialClock

Reply:
Changed int to duration. But I couldn't find standard TrivialClock concept implementation. Should I create it manually?